### ISSUE TITLE
Looks like /debug endpoint doesn't return the active jobs of RequesterJobs and ComputeJobs 

### ISSUE THREAD

**Issue by enricorotundo** [filecoin-project/bacalhau#1206]
```zsh
❯ curl -X POST 34.150.153.87:1234/debug | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   126  100   126    0     0    514      0 --:--:-- --:--:-- --:--:--   514
{
  "AvailableComputeCapacity": {
    "CPU": 3.2,
    "Memory": 12574726553,
    "Disk": 379498040524,
    "GPU": 1
  },
  "RequesterJobs": [],
  "ComputeJobs": []
}
❯ curl -X POST 35.245.115.191:1234/debug | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   119  100   119    0     0    506      0 --:--:-- --:--:-- --:--:--   506
{
  "AvailableComputeCapacity": {
    "CPU": 12.8,
    "Memory": 53944945868,
    "Disk": 409819358822
  },
  "RequesterJobs": [],
  "ComputeJobs": []
}
❯ curl -X POST 35.245.61.251:1234/debug | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   119  100   119    0     0    502      0 --:--:-- --:--:-- --:--:--   502
{
  "AvailableComputeCapacity": {
    "CPU": 12.8,
    "Memory": 53944952422,
    "Disk": 406550354329
  },
  "RequesterJobs": [],
  "ComputeJobs": []
}
❯ curl -X POST 35.245.251.239:1234/debug | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   119  100   119    0     0    495      0 --:--:-- --:--:-- --:--:--   495
{
  "AvailableComputeCapacity": {
    "CPU": 12.8,
    "Memory": 53944952422,
    "Disk": 408034020556
  },
  "RequesterJobs": [],
  "ComputeJobs": []
}
```

**Comment by aronchick** [comment]
@wdbaruni still valid?

**Comment by wdbaruni** [comment]
This is fixed by #1642 , but with separate endpoints for requester and debug info as follows:

```
→ curl 127.0.0.1:20000/requester/debug | jq
{
  "ScheduledJobs": [
    {
      "ShardID": "7f5989e2-06ac-4a4b-91f7-2205f6174508:0",
      "State": "WaitingForResults",
      "BiddingNodesCount": 1,
      "CompletedNodesCount": 0
    }
  ]
}


→ curl 127.0.0.1:20001/compute/debug | jq
  "ActiveJobs": [
    {
      "ExecutionID": "e-83b7097c-3283-47c0-ac11-04b0505041f5",
      "ShardID": "7f5989e2-06ac-4a4b-91f7-2205f6174508:0",
      "State": "BidAccepted",
      "ResourceUsage": {
        "CPU": 0.1,
        "Memory": 104857600
      }
    }
  ],
  "AvailableCapacity": {
    "CPU": 6.300000000000001,
    "Memory": 13639037747,
    "Disk": 189783736320
  }
}

```
